### DISCUSSION TITLE
[Feature] Asset download indicator/progress bar

### DISCUSSION THREAD

**Discussion by ryanrpj** [immich-app/immich#12974] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Currently, when downloading a large file, there is no way to know the download speed, ETA, nor progress. It would be nice to have a queue system for large file downloads and a page where it shows these infos. I've been in many situations that I just blindly waited for a 2GB+ file to finish downloading. Sometimes it just never finished at all, and I have no clue what happened.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#10100
